<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TouchAndControl2</class>
 <widget class="QMainWindow" name="TouchAndControl2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Touch And Control 2</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QPushButton" name="cierredia1">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>10</y>
      <width>113</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>cierre diario</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="busnom1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>124</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listBusq1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>156</y>
      <width>971</width>
      <height>91</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="resizeMode">
     <enum>QListView::Adjust</enum>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="gridSize">
     <size>
      <width>963</width>
      <height>44</height>
     </size>
    </property>
    <property name="itemAlignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="currentRow">
     <number>-1</number>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>104</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Busqueda por nombre</string>
    </property>
   </widget>
   <widget class="QPushButton" name="can1">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>120</y>
      <width>31</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="busnombut1">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>119</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>buscar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>556</x>
      <y>103</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Busqueda por Codigo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>523</x>
      <y>118</y>
      <width>31</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="buscod1">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>122</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
   </widget>
   <widget class="QPushButton" name="buscodbut1">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>118</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>buscar</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listDesglose1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>971</width>
      <height>261</height>
     </rect>
    </property>
    <property name="currentRow">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>260</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>true</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Desglose de venta</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="addcodven1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="placeholderText">
     <string>codigo de producto</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addlven1">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>271</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>añadir</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Añadir producto a venta</string>
    </property>
   </widget>
   <widget class="QPushButton" name="execvent1">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>270</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>procesar</string>
    </property>
   </widget>
   <widget class="QTableView" name="tabdes1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>971</width>
      <height>261</height>
     </rect>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="cantprodven1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>270</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="placeholderText">
     <string>cantidad</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tabwiddesg1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>971</width>
      <height>261</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>5</number>
    </property>
    <column>
     <property name="text">
      <string extracomment="codigo del producto">codigo</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>nombre</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>precio</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>cant.</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>total</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>560</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Venta total ($): </string>
    </property>
   </widget>
   <widget class="QLabel" name="ventot1">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>560</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly|Qt::ImhPreferNumbers</set>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="bg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1031</width>
      <height>601</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>NOT TRANSPARENT-13.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="elifrdes1">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>560</y>
      <width>31</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>x</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <zorder>bg</zorder>
   <zorder>cierredia1</zorder>
   <zorder>busnom1</zorder>
   <zorder>listBusq1</zorder>
   <zorder>label_2</zorder>
   <zorder>can1</zorder>
   <zorder>busnombut1</zorder>
   <zorder>label_3</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>buscod1</zorder>
   <zorder>buscodbut1</zorder>
   <zorder>listDesglose1</zorder>
   <zorder>label_4</zorder>
   <zorder>addcodven1</zorder>
   <zorder>addlven1</zorder>
   <zorder>label_5</zorder>
   <zorder>execvent1</zorder>
   <zorder>tabdes1</zorder>
   <zorder>cantprodven1</zorder>
   <zorder>tabwiddesg1</zorder>
   <zorder>label_6</zorder>
   <zorder>ventot1</zorder>
   <zorder>elifrdes1</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
